pkg/web/templates/partials/password/password_changed.html

```diff
@@ -0,0 +1 @@
+<div></div>
```

Comment:
A swap didn't seem necessary here, but this file had to be included to prevent a panic.